src/ExpressionEvaluator/Core/Source/ExpressionCompiler/MetadataUtilities.cs

```diff
@@ -178,9 +175,24 @@ internal static string GetFileNameWithoutExtension(string fileName)
             return fileName;
         }
 
+        private static byte[] GetWindowsProxyBytes()
+        {
+            var assembly = typeof(ExpressionCompiler).GetTypeInfo().Assembly;
```

Comment:
Once I heard a scary story about a bug that was caused by cross-assembly inlining of a method containing `Assembly.GetExecutingAssembly()` invocation.